src/components/dashboard/dividerStack.js

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import Divider from '@mui/material/Divider'
+import Typography from '@mui/material/Typography'
+import Paper from '@mui/material/Paper'
+import Stack from '@mui/material/Stack'
+import { styled } from '@mui/material/styles'
```

Comment:
No need to import react like this
```suggestion
import Divider from '@mui/material/Divider'
import Typography from '@mui/material/Typography'
import Paper from '@mui/material/Paper'
import Stack from '@mui/material/Stack'
import { styled } from '@mui/material/styles'
```